idd/Energy+.idd.in

```diff
@@ -45192,7 +45201,11 @@ Coil:Cooling:DX:TwoStageWithHumidityControlMode,
   A14, \field Condensate Collection Water Storage Tank Name
        \type object-list
        \object-list WaterStorageTankNames
-  N5,  \field Basin Heater Capacity
+  N5,  \field Minimum Outdoor Dry-Bulb Temperature for Compressor Operation {C}
```

Comment:
Different OAT cutout for different stages? It's here because there would only be one minimum temp. I guess it's possible to have a different cutout for different stages.